relay-event-schema/src/protocol/span.rs

```diff
@@ -76,6 +76,14 @@ pub struct Span {
     /// Timestamp when the span has been received by Sentry.
     pub received: Annotated<Timestamp>,
 
+    /// Measurements which holds observed values such as web vitals.
+    ///
+    /// Measurements are only available on transactions. They contain measurement values of observed
+    /// values such as Largest Contentful Paint (LCP).
+    #[metastructure(skip_serialization = "empty")]
+    #[metastructure(omit_from_schema)] // we only document error events for now
```

Comment:
```suggestion
```

Not sure if this is necessary, we don't set it on the other fields in this struct.

Reply:
This is necessary since the `JSONSchema` trait is not added on `Measurements`.